blog-service/src/main/java/com/example/blogservice/service/CategoryService.java

```diff
@@ -0,0 +1,47 @@
+package com.example.blogservice.service;
+
+import com.example.blogservice.dto.CategoryRequest;
+import com.example.blogservice.dto.CategoryResponse;
+import com.example.blogservice.model.CategoryModel;
+import com.example.blogservice.repository.ICategoryRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class CategoryService {
+
+
+    private final ICategoryRepository categoryRepository;
+
+    public void createCategory(CategoryRequest categoryRequest) {
+        CategoryModel category = CategoryModel.builder()
```

Comment:
In my opinion, I prefer to avoid using `lombok Builder`. It causes unexpected bugs